CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#1698](https://github.com/CocoaPods/CocoaPods/issues/1698) 
 
+* Prevents need for .swift-version file in Objective-C pods  
+ +  [Austin Emmons](https://github.com/atreat)
```

Comment:
I think it has to do with formatting. I was commenting mostly that the second line was not aligned with the top line.